### ISSUE TITLE
client.aggregate_trade_iter gives back error

### ISSUE THREAD

**Issue by thePapan** [sammchardy/python-binance#341]
Hi,

I'm using the client.aggregate_trade_iter function and it sometimes yields the following error: 

> 
>   File "<ipython-input-1-b58303995740>", line 210, in get_trades
>     trades_list=list(client.aggregate_trade_iter(symbol=self.ticker,start_str='2 minutes ago UTC'))
> 
>   File "/usr/local/lib/python2.7/dist-packages/binance/client.py", line 633, in aggregate_trade_iter
>     last_id = trades[-1][self.AGG_ID]
> 
> IndexError: list index out of range

Any idea why ?

Thanks

**Comment by sapph1re** [comment]
Same issue

**Comment by sapph1re** [comment]
Apparently it happens when I hit the API limits. Binance client doesn't handle it properly and simply crashes with an IndexError.

**Comment by sapph1re** [comment]
It happens when the server responds with an empty list (i.e. there have been no trades).
```
HTTP status: 200
Response body: []
```
The library doesn't process this empty list well and crashes on line 633.

**Comment by sapph1re** [comment]
Actually, the comments in the code even mention that:
```
                # The difference between startTime and endTime should be less
                # or equal than an hour and **the result set should contain at**
                # **least one trade**.
```
But the code doesn't handle the case when the result set doesn't contain any trades.
Users of this library can't guarantee that every set will contain at least one trade, so I believe it's the library that should take care of this.

**Comment by sammchardy** [comment]
Fixed in version 0.7.1 thanks.
